python/cuml/test/test_nearest_neighbors.py

```diff
@@ -198,7 +202,7 @@ def test_return_dists():
 
 @pytest.mark.parametrize('input_type', ['dataframe', 'ndarray'])
 @pytest.mark.parametrize('nrows', [unit_param(500), quality_param(5000),
```

Comment:
If the stress test is the same as the quality test, we should probably not have both. Can we increase to 50,000 so it is still really stressing the ability to scale? Getting above 2**16 is always nice. I would probably remove the quality test param (which is rarely used) but try to increase the stress test beyond 5k. That would be only a few MB of data, so not too stressful.

Reply:
True, I augmented this parameter to 70k to get above 2**16